README.md

```diff
@@ -39,251 +50,6 @@ For docker, please refer to the [Dockerfile] and [Docker Compose Builder] script
 [Dockerfile]: Dockerfile
 [Docker Compose Builder]: https://github.com/edgexfoundry/edgex-compose/tree/main/compose-builder
```

Comment:
do we want to link to the latest release of compose builder instead of main?

Reply:
We don't have a macro for that from the repo readme, so it has to be main

Reply:
Note on the Releases we create we do point to the version specific Compose Builder.